### ISSUE TITLE
Refresh Button In Devices Packages Does not Work

### ISSUE THREAD

**Issue by LeoNerdoG** [eclipse/kapua#1049]
Refresh button in Packages under Devices does not work. If user (un)installs a package, it has to refresh the whole page or select any other device/tab/... and then return to this device. After this, Package list is refreshed. 

Testflow: 
1. Login as kapua-sys
2. Go to Users, create a user (e.g. kapua-user1)
3. Go to Devices and add a live device
4. Select kapua-user1, add **_read_** permissions to devices, and _**ALL**_ permissions for device_management domain
5. Login as kapua-user1, go to Devices, select sample-device1 and go to Packages tab
6. Try to download package from Eclipse Marketplace (e.g. example publisher)
**Package gets downloaded and installed successfully, but Packages tab is not refreshed. If user presses _Refresh_ button, list is not refreshed.**

![screen shot 2017-11-23 at 09 12 54](https://user-images.githubusercontent.com/15121999/33163553-5b639806-d02f-11e7-872c-f91feb3f3d26.png)

Browsers used:
1. Google Chrome (Version 62.0.3202.94 (Official Build) (64-bit))
2. Safari (Version 11.0.1 (13604.3.5))
3. Mozilla Firefox (57.0 (64-bit))
4. Opera (Version 49.0.2725.39)

**Comment by muros-ct** [comment]
Investigating this issue, I came to solution to enable Refresh button on Installed Packages.
Problem is that when new package is installed it is not installed as quick as action is completed and consequently it does not get into list of installed packages before async action of installation is completed.
This issue could be different form package to package, depending on how fast it is installed on Kura device. Because of this refresh is only viable solution.
I enabled refresh on Installed Packages and it works.
